library/Freckle/App/Database.hs

```diff
@@ -199,6 +200,7 @@ envParseDatabaseConf source = do
   database <- Env.var Env.nonempty "PGDATABASE" mempty
   port <- Env.var Env.auto "PGPORT" mempty
   poolSize <- Env.var Env.auto "PGPOOLSIZE" $ Env.def 10
+  schema <- Env.var Env.auto "PGSCHEMA" mempty
```

Comment:
`auto` means `Read a => a`; it's good for numeric types. Here, you're using it with `a ~ Maybe String`. In other words, this parser will accept `PGSCHEMA=Just "Something"'` or `PGSCHEMA=Nothing`, which is not what you want.

When you're parsing a text type, there's no reason to `Read` it, you can just use `str`. In this case though, we don't want the variable to be empty either, so I'd use `nonempty`. Getting it to `Maybe` is done via `optional`.

```suggestion
  schema <- optional (Env.var Env.nonempty "PGSCHEMA" mempty)
```